src/network/hooks/useSponsorshipsList.ts

```diff
@@ -94,26 +101,36 @@ export const useMySponsorshipsQuery = (
     })
 }
 
-export const mapSponsorshipToElement = (sponsorship: Sponsorship): SponsorshipElement => {
+export const mapSponsorshipToElement = (
+    sponsorship: Sponsorship,
+    decimals: number,
```

Comment:
`NaN` decimals will make anything that relies on it `NaN`, too. Also, remember that `typeof NaN` is `number`.